lib/cc/cli/analyze.rb

```diff
@@ -1,6 +1,17 @@
+require "cc/cli/command"
+
 module CC
   module CLI
     class Analyze < Command
+      ARGUMENT_LIST = "[-f format] [-e engine[:channel]] [path]".freeze
+      SHORT_HELP = "Run analysis with the given arguments".freeze
+      HELP = "#{SHORT_HELP}\n" \
+        "\n" \
+        "    -f <format>, --format <format>   Format of output. Possible values: #{CC::Analyzer::Formatters::FORMATTERS.keys.join ", "}\n" \
+        "    -e <engine[:channel]>            Engine to run. Can be specified multiple times.\n" \
+        "    --dev                            Run in development mode.\n" \
```

Comment:
Might be good to expand on this. WDYT of "Run in development mode. Engines installed locally that are not in the manifest will be run"